bftengine/src/bcstatetransfer/AsyncStateTransferCRE.cpp

```diff
@@ -71,15 +72,7 @@ class Communication : public ICommunication {
   uint16_t repId_;
 };
 
-class InternalSigner : public concord::crypto::ISigner {
- public:
-  size_t signBuffer(const concord::Byte* dataIn, size_t dataLen, concord::Byte* sigOutBuffer) override {
-    return bftEngine::impl::SigManager::instance()->sign(dataIn, dataLen, sigOutBuffer);
-  }
-
-  size_t signatureLength() const override { return bftEngine::impl::SigManager::instance()->getMySigLength(); }
-  std::string getPrivKey() const override { return ""; }
-};
+// Implement cryptomanager update handler
```

Comment:
I guess you mean to the MainKeyUpdateHandler, if so, please remove the comment. If not, then what does the comment means?

Reply:
removed